src/tests/particles/artifacts/products-test-ng.recipes

```diff
@@ -0,0 +1,58 @@
+// @license
```

Comment:
nit - this file should be `.arcs` and upper camel case (ProductsTestNg.arcs) - i know this is inconsistent with the older file, but that one predates the convention :)

Reply:
Done